Build broken on 0.13.0

I'm unable to build o2w with my system Zig binary from what appears to be a signature change:
```
/home/user/.cache/zig/p/12200e5959fd68f7b2101eae5b6f23a213712810814cef42f44e112cc7b4f14f0248/build.zig:20:33: error: no field named 'path' in union 'Build.LazyPath'
        .root_source_file = .{ .path = "src/lib.zig" },
                                ^~~~
/usr/lib/zig/std/Build.zig:2171:22: note: union declared here
pub const LazyPath = union(enum) {
                     ^~~~~
referenced by:
    runBuild__anon_16343: /usr/lib/zig/std/Build.zig:2116:27
    dependencyInner__anon_15130: /usr/lib/zig/std/Build.zig:2097:29
    remaining reference traces hidden; use '-freference-trace' to see all reference traces
/home/user/.cache/zig/p/12206275494db106adf893d237dda64130436e36cb91b60ddc960bd50b938690a8dc/build.zig:8:33: error: no field named 'path' in union 'Build.LazyPath'
        .root_source_file = .{ .path = "src/root.zig" },
                                ^~~~
/usr/lib/zig/std/Build.zig:2171:22: note: union declared here
pub const LazyPath = union(enum) {
                     ^~~~~
/home/user/.cache/zig/p/1220949d4e88864579067b6d4cdad6476c6176f27e782782c2c39b7f2c4817a10efb/build.zig:4:69: error: no field named 'path' in union 'Build.LazyPath'
    const clap_mod = b.addModule("clap", .{ .root_source_file = .{ .path = "clap.zig" } });
                                                                    ^~~~
/usr/lib/zig/std/Build.zig:2171:22: note: union declared here
pub const LazyPath = union(enum) {
                     ^~~~~
/home/user/.cache/zig/p/1220e7e5cb6837d9b0bc9b11f99d7a0660ea89490c41adcfede311196d9dc7e14066/build.zig:7:35: error: no field named 'path' in union 'Build.LazyPath'
    _ = copyFiles.addCopyFile(.{ .path = "config.h.generic" }, "config.h");
                                  ^~~~
/usr/lib/zig/std/Build.zig:2171:22: note: union declared here
pub const LazyPath = union(enum) {
                     ^~~~~
```

Reproducible with a 0.13.0 build from ziglangorg, doesn't appear with a 0.12.0 build.

--- a/.gitignore
+++ b/.gitignore
@@ -6,3 +6,4 @@ src/entities.zig
 deps.zig
 public/
 *.o2w
+.zig-cache/

--- a/README.md
+++ b/README.md
@@ -15,8 +15,8 @@ i learned this the hard way. one day i'll fix it.)
 
 # installation
 
-- get a recent master build off https://ziglang.org/download/
-  - tested with `0.12.0`
+- get a recent build off https://ziglang.org/download/
+  - tested with `0.13.0`
 - install libpcre in your system
 
 ```

--- a/build.zig.zon
+++ b/build.zig.zon
@@ -4,20 +4,20 @@
     .minimum_zig_version = "0.12.0",
     .dependencies = .{
         .@"libpcre.zig" = .{
-            .url = "https://github.com/kivikakk/libpcre.zig/archive/0f3e120c75c297f7d88a5c698d941a9e730a2059.tar.gz",
-            .hash = "12208c8c5b46df969207ca1458b95ebbe988892d1f0e028d1fb95bd58cf55bca6628",
+            .url = "git+https://github.com/kivikakk/libpcre.zig#1dc32407368651bc0118b752288c3b87af12ff50",
+            .hash = "1220039ac1b212be156069f9ed7e066571a8da5acc8f1adc33eb1a5c3bcbb4a8b47b",
         },
         .chrono = .{
-            .url = "https://github.com/leroycep/chrono-zig/archive/bedd9339af9b05c240956b137cb1a5b67a9b74b7.tar.gz",
-            .hash = "12200e5959fd68f7b2101eae5b6f23a213712810814cef42f44e112cc7b4f14f0248",
+            .url = "git+https://github.com/lun-4/chrono-zig#56b4ea68c06c79988d3f0971be8772cffb73706e",
+            .hash = "1220449e26ca29770522d4e46c5147b126ca4f149d257dc58e4ceca217f7bbfd8090",
         },
         .@"zig-uuid" = .{
-            .url = "https://github.com/nitrogenez/zig-uuid/archive/4fc94de547b84b5db6d93cf50958317b6014e561.tar.gz",
-            .hash = "12206275494db106adf893d237dda64130436e36cb91b60ddc960bd50b938690a8dc",
+            .url = "git+https://github.com/lun-4/zig-uuid#a269818e9b3d503be69319ce8233609bf7f78fe8",
+            .hash = "12202f571bbf695f4c78306182baee0de40f532fba2d2f6cc14e7b81fbf7614f9baf",
         },
         .koino = .{
-            .url = "https://github.com/kivikakk/koino/archive/26b38766c96fc44757f45db8bddf8d57fbca09a4.tar.gz",
-            .hash = "122078d4bb0c4c9a5fdc6293c480a8d40ee544dab7ba3919bd11e7342ca55ff87f6a",
+            .url = "git+https://github.com/kivikakk/koino#0151bb37714d93688f31e3d7a3d0369106818f26",
+            .hash = "122055057fe62c0a8bcccb3c36a23115db58a185af2c18dd41d0bb4ac083f519ad5d",
         },
     },
     .paths = .{

--- a/src/main.zig
+++ b/src/main.zig
@@ -1197,7 +1197,7 @@ fn toRFC822(allocator: std.mem.Allocator, timestamp: i64) ![]const u8 {
     defer env_map.deinit();
     try env_map.put("TZ", "UTC");
 
-    const result = try std.ChildProcess.run(.{
+    const result = try std.process.Child.run(.{
         .allocator = allocator,
         .argv = argv.items,
         .max_output_bytes = 256,

--- a/src/processors.zig
+++ b/src/processors.zig
@@ -532,7 +532,7 @@ pub const CodeblockProcessor = struct {
             "/tmp/o2w_sex2",
         });
 
-        const result = try std.ChildProcess.run(.{
+        const result = try std.process.Child.run(.{
             .allocator = ctx.allocator,
             .argv = argv.items,
             .max_output_bytes = 100 * 1024,
@@ -693,7 +693,7 @@ pub const StaticTwitterEmbed = struct {
 
         var file = dir.openFile(pathname, .{ .mode = .read_only }) catch |err| switch (err) {
             error.FileNotFound => blk: {
-                var proc = std.ChildProcess.init(
+                var proc = std.process.Child.init(
                     &[_][]const u8{
                         "snscrape",
                         "-vv",

